giraffe/src/utils/useTooltipStyle.ts

```diff
@@ -52,6 +52,7 @@ export const useTooltipStyle = (el: HTMLDivElement) => {
       position: 'fixed',
       left: `${clampedX}px`,
       top: `${clampedY}px`,
+      zIndex: 400,
```

Comment:
This seems like an arbitrary value. Why are we using 400 for the z-index?

Reply:
It is because the library (leaflet) we use to render maps is rendering the map tiles into an absolutely positioned element with z-index 400. Tooltips then appear under the map and are not visible. Ihave added some comments into the code to explain this.